packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -57,6 +57,34 @@
                                     data-translate="pci_projects_project_quota_threshold_warning_message"
                                     data-translate-values="{ quotaUrl: $ctrl.getQuotaUrl() }"
                                 ></p>
+                                <p></p>
+                            </oui-message>
+                            <oui-message
+                                data-type="info"
+                                data-ng-if="$ctrl.isDiscoveryProject"
+                            >
+                                <div class="font-weight-bold">
+                                    <span
+                                        data-translate="pci_projects_project_discovery_banner_title_1"
+                                    ></span>
+                                    <span
+                                        class="promotion-value"
+                                        data-translate="pci_projects_project_discovery_banner_title_2"
+                                    >
+                                    </span>
+                                </div>
```

Comment:
**nitpick:** 
```suggestion
                                <h6>
                                    <span
                                        data-translate="pci_projects_project_discovery_banner_title_1"
                                    ></span>
                                    <span
                                        class="promotion-value"
                                        data-translate="pci_projects_project_discovery_banner_title_2"
                                    >
                                    </span>
                                </h6>
```

Reply:
Additionnally, I added a `m-0` css class to the <h6> to fit the UI Mockup